### DISCUSSION TITLE
Fan Contro after close

### DISCUSSION THREAD

**Discussion by carlosyrodriguez** [Rem0o/FanControl.Releases#804] [General]
After closing app and trying to open it, the app doesn't open and I have to keep attempting to open (spamming app icon on hidden apps is temp fix), hoping for a fix.

Also would be great if the fan control config had the ability to run in the background after boot instead of opening the app window

**Comment by Rem0o** [comment]
Just to be sure, the app works as a background app, but has a UI available in the tray area.  "X" ing out the window does NOT close the app. A tray icon will remain. You must click the "exit" button from the 3dot menu or from the tray icon for the app to close. You can't reopen the window by double clicking the icon on your desktop.

**Reply by carlosyrodriguez** [reply]
Ohh, user error it is, thank you!

**Comment by xhul-dev** [comment]
> Also would be great if the fan control config had the ability to run in the background after boot instead of opening the app window

There's already a "Start with windows" option available in the app, aswell as "Start minimized".
Those two should solve your problem.

**Reply by carlosyrodriguez** [reply]
Thank you!
